tests/gtest/aviflosslesstest.cc

```diff
@@ -64,26 +65,70 @@ void ReadImageSimple(const std::string& file_name, avifPixelFormat pixel_format,
   ASSERT_NE(file_format, AVIF_APP_FILE_FORMAT_UNKNOWN);
 }
 
+// Checks whether an image is grayscale. ASSERT_NO_FATAL_FAILURE should be used
+// when calling it.
+void IsGray(const std::string& path, bool& is_gray) {
+  ImagePtr image(avifImageCreateEmpty());
+  ASSERT_NE(image, nullptr);
+  // In the first iteration, figure out if the image is grayscale.
+  image->matrixCoefficients = AVIF_MATRIX_COEFFICIENTS_UNSPECIFIED;
+  ASSERT_NE(AVIF_APP_FILE_FORMAT_UNKNOWN,
+            avifReadImage(
+                path.c_str(), AVIF_PIXEL_FORMAT_NONE, /*requested_depth=*/0,
+                /*chroma_downsampling=*/AVIF_CHROMA_DOWNSAMPLING_AUTOMATIC,
```

Comment:
All of the parameters to this function are lower camel case. The comments need an update.

Reply:
Done